app/Models/taskcrudapp.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class taskcrudapp extends Model
```

Comment:
Class name should be starting with Caps following each word in caps. In this case it will be `TaskCrudApp`.
Model name should be `Task` as below:
```
class Task extends Model
```